coffeelint.json

```diff
@@ -27,5 +27,17 @@
   "braces_spacing": {
     "spaces": 0,
     "level": "error"
+  },
+  "space_operators": {
```

Comment:
This one I'm not a huge fan of since it picks up things like:

```
setFullScreen: (fullScreen=false) ->
```

The contributing guide mentions no spaces around param defaults, https://github.com/atom/atom/blob/master/CONTRIBUTING.md#coffeescript-styleguide